02.calendar/calendar.js

```diff
@@ -0,0 +1,45 @@
+const minimist = require('minimist')
+const { DateTime } = require('luxon')
+
+const date = DateTime.local()
+
+const argv = minimist(process.argv.slice(2), {
+  string: ['output'],
+  default: {
+    m: date.month,
+    y: date.year
+  }
+})
+
+const inputDate = DateTime.fromObject({
+  year: argv.y,
+  month: argv.m
+})
+
+const lastDay = inputDate.daysInMonth
+const firstWeekday = inputDate.weekday
+
+console.log('    ', argv.m, '月', argv.y, '年')
+console.log(' 日 月 火 水 木 金 土')
```

Comment:
```suggestion
console.log('日 月 火 水 木 金 土')
```

のように先頭にスペースが入らない形でフォーマットできるとより良いですね。

Reply:
こちらも修正させていただきました。